common/djangoapps/util/course.py

```diff
@@ -76,7 +76,15 @@ def has_certificates_enabled(course):
     return course.cert_html_view_enabled
 
 
-def should_display_grade(end_date):
-    if end_date and end_date < now().replace(hour=0, minute=0, second=0, microsecond=0):
-        return True
-    return False
+def should_display_grade(course_overview):
+    """
+    Returns True or False depending upon either certificate available date
+    or course-end-date
+    """
+    course_end_date = course_overview.end_date
+    cert_available_date = course_overview.certificate_available_date
+    current_date = now().replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
Shouldn't we use UTC current time here?

Reply:
here, I'm now() func from django utils which is taking care of utc.
https://github.com/django/django/blob/master/django/utils/timezone.py#L188